workflow/rocoto/gfs_tasks.py

```diff
@@ -1513,7 +1525,7 @@ def gempakmetancdc(self):
 
     def gempakncdcupapgif(self):
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.cdump}gempak'}
+        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}gempak'}
```

Comment:
Same comment as for the gempakmeta job, no change was made for the gempakncdcupapgif job.